There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs

```diff
@@ -14,12 +12,9 @@ class Player
 {
         static void Main(string[] args)
         {
-          
             // ### Get Input Details
-
-            /* <p><i>Get input details includign width, height, and the lines. Use horizontal and vertical lines to store the input details for processing</i></p>
+            /* <p><i>Get input details including width, height, and the lines. Use horizontal and vertical lines to store the input details for processing</i></p>
```

Comment:
Remove HTML, rewrite more simply using Markdown.

Reply:
The issue is that styling was an issue is some cases with markdown. Using HTML seemed to be quicker. I couldn't find a way to resize the image in the solution while improving structuring of the page. Suggestions for resizing the image were to include a CSS file, but that didn't seem to work well with CodeChat. Are CSS files supported in CodeChat?

Reply:
I fixed the issue buy resizing the image before adding it to the document.